### DISCUSSION TITLE
抖音小程序是否支持？

### DISCUSSION THREAD

**Discussion by AxingG** [Tencent/libpag#2129] [Q&A] [answered]
我看到PAG有支持Flutter、微信小程序，但是目前uni-app 多端平台基本共用一套代码，想问下PAG是否支持除了微信之外的其他小程序？

**Comment by kevingpqi123** [comment, marked as answer]
暂时没有计划支持，我们没有这样的应用场景，pag 是开源的，欢迎参与共建
